src/Microsoft.VisualStudio.ProjectSystem.Managed/Tree/Dependencies/DependenciesProjectTreeProvider.cs

```diff
@@ -454,7 +454,7 @@ private async Task OnDependenciesSnapshotChangedAsync(SnapshotChangedEventArgs e
 
                 if (!cancellationToken.IsCancellationRequested)
                 {
-                    dependenciesNode = await viewProvider.BuildTreeAsync(dependenciesNode, snapshot, cancellationToken);
+                    dependenciesNode = await viewProvider!.BuildTreeAsync(dependenciesNode, snapshot, cancellationToken);
```

Comment:
This seems like a regression in the compiler.

Reply:
I was unsure about that, @agocke @jcouv Thoughts?

Null checked here: https://github.com/dotnet/project-system/blob/8f558fe5d4f6891164601017b48d15a7bdb83ea6/src/Microsoft.VisualStudio.ProjectSystem.Managed/Tree/Dependencies/DependenciesProjectTreeProvider.cs#L427
Had to `!` here: https://github.com/dotnet/project-system/blob/8f558fe5d4f6891164601017b48d15a7bdb83ea6/src/Microsoft.VisualStudio.ProjectSystem.Managed/Tree/Dependencies/DependenciesProjectTreeProvider.cs#L457.